.uncrustify.cfg

```diff
@@ -17,6 +18,19 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+#
+# This program is free software; you can redistribute it and/or modify it under the terms of the
```

Comment:
The way the license snippets are concatenated together, it looks like you have to obey _both_ licenses to use the program. This is genuinely unclear, and creates a legal risk for users.

Having only the SPDX license identifier and none of the rest of the boilerplate would be unambiguous. Do we have a requirement to include the rest of the boilerplate in each file?

Do we have some corporate guidance on how to write a dual license? Failing that, does a reputable FOSS-related organization offer such guidance?

Reply:
I will check this. I agree it's a bit confusing.

Reply:
The OSO guidance is: `The recommended license notice is the [SPDX License Identifier](https://spdx.org/licenses/). It is also acceptable to include any short license notice as defined by the license itself. Apache 2.0 and the GPL family are examples of licenses that define their own short license notice`.

So having only the SPDX license identifier would be acceptable, and I think better for readability & unambiguity.